tools/install.sh

```diff
@@ -77,11 +77,23 @@ case $DISTRO_NAME in
   sudo dnf install /tmp/OpenBangla.rpm
   ;;
 (*)
-  echo
-  echo "Sorry, this script was not able to identify your OS distribution."
-  echo
-  echo "Please consult https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/"
-  echo "for distrowise install or build instructions."
-  echo
-  ;;
+      echo
+      echo "This script was not able to identify your OS distribution! Please report a bug at https://github.com/OpenBangla/OpenBangla-Keyboard/issues"
+      echo
+      echo "Please visit https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Installing-OpenBangla-Keyboard for Distrowise/Distro-specific Install Instructions."
```

Comment:
I think it should be `install instructions`

"Please visit https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Installing-OpenBangla-Keyboard for Distrowise/Distro-specific install instructions."